lib/dataset_augmenter.js

```diff
@@ -44,6 +45,7 @@ module.exports = class DatasetAugmenter extends Stream.Transform {
             noQuoteExpandFactor: this._options.noQuoteExpandFactor,
             paraphrasingExpandFactor: this._options.paraphrasingExpandFactor,
             replaceLocations: this._options.replaceLocations,
+            replaceNumbers: this._options.replaceNumbers,
```

Comment:
This is going to be tricky to implement by the way, because NUMBER_ cannot always be replaced with SLOT_ tokens (which happens as a prereq to loading the program so you can typecheck & iterate the slots). It's mostly a problem around measurements, IIRC.

Reply:
True that. I am only changing the requote and augment.
We will always have numbers as placeholders elsewhere. The numbers are replaced JUST before translation and then requoted RIGHT after translation.